index.html

```diff
@@ -282,6 +282,57 @@ <h5>Skills</h5>
           </ul>
         </div>
       </section>
+
+      <section id="contact-section">
+        <div class="contact">
+          <h6>Contact me</h6>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. <br />
+            I’d love to help with it!
+          </p>
+          <form
+            action="https://formspree.io/f/xrgvjnrg"
+            class="form"
+            method="post"
+          >
+            <div class="form-group">
+              <input
+                class="types"
+                type="text"
+                id="name"
+                placeholder="Full Name"
+                maxlength="30"
+                required
+              />
+            </div>
+            <div class="form-group">
+              <input
+                class="types"
+                type="email"
+                id="email"
+                placeholder="jameshard@gmail.com"
+                required
+              />
+            </div>
+            <div class="form-group">
+              <textarea
```

Comment:
[OPTIONAL]
- [ ] please can your style the color of the textarea as shown from your design below it is not so clear once a user starts to type in their details please can you ensure to make this part as clear as can be. Thsnk you :+1: 
- your design
![nv](https://github.com/alyconr/Personal-Portfolio-Project/assets/89055502/485a5ed2-ee6d-44b9-af65-ca3ce236bfae)